src/client/app/app.react.js

```diff
@@ -94,7 +96,7 @@ class App extends Component {
         <RouteHandler {...this.state} />
         <footer>
           <p>
-            made by <a href="https://twitter.com/steida">steida</a>
+            <FormattedHTMLMessage message={msg('app.madeByHtml')} />
```

Comment:
Well, by my experience you always need 3 messages if you want plain text in translation and using one tag at the same time. Different languages - different order of words. Then there are languages like Greek or Hungarian with very long words compared to english. Coz of the different order of words you end with putting completely unrelated words under given placeholders. And it's much easier to explain to translator not to delete tags than explain coder not to mess the component, if he has to hardcode some language specific text. But it would be highly appreciated if you find better solution. Theoretically is having tag in translation awful, practically it's amazing, especially while using <br> and more placeholders in texts for PDF tax documents. :-)